x/concentrated-liquidity/types/keys.go

```diff
@@ -22,10 +21,13 @@ const (
 
 // Key prefixes
 var (
-	TickPrefix      = []byte{0x01}
-	PositionPrefix  = []byte{0x02}
-	PoolPrefix      = []byte{0x03}
-	IncentivePrefix = []byte{0x04}
+	TickPrefix           = []byte{0x01}
+	PositionPrefix       = []byte{0x02}
+	PoolPrefix           = []byte{0x03}
+	IncentivePrefix      = []byte{0x04}
+	PositionIdPrefix     = []byte{0x08}
+	PoolPositionPrefix   = []byte{0x09}
+	FeeAccumulatorPrefix = []byte{0x0A}
```

Comment:
Is there a reason we're skipping keys for 5,6,7?
```suggestion
	PositionIdPrefix     = []byte{0x05}
	PoolPositionPrefix   = []byte{0x06}
	FeeAccumulatorPrefix = []byte{0x07}
```

Reply:
So this is why PositionIdPrefix starts at 8, because we last ended at 7 for KeyNextGlobalPositionId